### PR TITLE
make sure pm2 home is set for pm2 list

### PR DESCRIPTION
**Pull request by jarosser06**

### CODE REVIEW

File: providers/application.rb [hunk 1]
```diff
@@ -130,7 +130,10 @@ def pm2_command(pm2_command)
 end
 
 def pm2_app_online?
-  cmd = shell_out!('pm2 list', :user => new_resource.user, :returns => 0)
+  cmd = shell_out!('pm2 list',
+                   :user => new_resource.user,
+                   :environment => { 'PM2_HOME' => new_resource.home },
```
Comment (miguelcnf) [inline providers/application.rb]: We need to check for new_resource.home.nil? here?

Comment (jarosser06) [inline providers/application.rb]: Good catch.  I'll add it

**Comment by jarosser06** [comment]
Fix for the environment added

**Comment by miguelcnf** [comment]
Released [0.3.3](https://github.com/Mindera/pm2-cookbook/releases/tag/0.3.3) with this - thanks.

**Pull request merged**